src/components/accession2/create/Accession2Create.tsx

```diff
@@ -65,6 +67,23 @@ export default function CreateAccession(): JSX.Element {
 
   const [record, setRecord, onChange] = useForm<AccessionPostRequestBody>(defaultAccession());
 
+  const { availableProjects } = useProjects(record);
```

Comment:
just `useProjects()` since we aren't leveraging anything from the record?

Reply:
Good call